composer.json

```diff
@@ -16,12 +16,19 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0",
-        "guzzlehttp/guzzle": "~6.0"
+        "ext-json": "*",
+        "php": ">=7.1",
```

Comment:
It looks like this PR is plant to drop `php-5.x` and `php-7.0` supports.

And just notice that we should drop these PHP versions in `.travis.yml` setting.